synapse/rest/client/v1/login.py

```diff
@@ -370,28 +374,49 @@ def do_jwt_login(self, login_submission):
         defer.returnValue(result)
 
 
-class CasRedirectServlet(RestServlet):
+class BaseSsoRedirectServlet(RestServlet):
```

Comment:
Shouldn't it be named `BaseSSORedirectServlet` because of SSO being uppercase in the abbreviation form?

Reply:
meh, maybe. I find `BaseSsoRedirectServlet` more readable.

Reply:
I suppose we do have an SSOAuthHandler. I'll change it.